packages/ts-api/src/types/artists.ts

```diff
@@ -0,0 +1,44 @@
+//If updating types within the types folder please make sure to do so in both the next-frontend and ts-api workspaces;
+export interface ArtistProfile {
```

Comment:
Mmh, shall we somehow share these types between API and frontend? I makes totally sense to have 2 files as it is right now, because one lives in the frontend and one in the backend, but I think it could save us some effort during development if they would be one (no confusion about which file you're currently adding properties to, no missing properties etc.).

Reply:
If you're leaning towards keeping them separated I'd vote for using different types names.

Reply:
I would like to as well. I did a quick search on sharing files between yarn workspaces and wasn't able to find anything helpful. If you know something or can find something I missed I would love to consolidate these into one folder.

Reply:
What do you think about creating a shared workspace / package:

- packages/
  - next-frontend/
      - package.json
  - ts-api/
      - package.json
  - api-types/ (or similar)
      - src/
        - Types.tsx

Reply:
package.json could like this:
```
{
 ...
  "dependencies": {
    "api-types": "1.0.0",
  }
}

```

Reply:
Yeah, `types` sounds good.

Reply:
So as far as I can tell the best way to do this would be to publish an npm package with our types. On face value it doesn't seem that challenging but I'm inclined to hold off and make that its own ticket so that we can put some extra thought into it instead of rushing to do it to complete this ticket.